Slack.Automation/Promact.Erp.DomainModel/Models/Group.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Promact.Erp.DomainModel.Models
+{
+    public class Group
```

Comment:
Please Inherit ModelBase class for adding Id(Primary key) and CreatedDate. 
Same as all domain model class